Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -486,6 +486,11 @@ public final class PackageInfoMapper: PackageInfoMapping {
                 do {
                     let condition = try ProjectDescription.PlatformCondition.from(setting.condition)
```

Comment:
This doesnt throw anymore, so we should be able to update this callsite and remove the `do {} catch {}` also

Reply:
oh yeah, good point